src/components/images_uploading/DisplayImageList.js

```diff
@@ -44,13 +44,41 @@ class UserAssignmentDropDown extends React.Component {
 }
 
 class DisplayImageList extends Component {
+  constructor(props) {

```

Comment:
Now we can 'delete apartment' and after we delete any apartment, the page will automatically refresh.
The code can definitely be improved, but at least it's working now (I'll see if I can make it better later T^T)